components/TabLinks.vue

```diff
@@ -10,6 +10,8 @@
           :to="link.href"
           class="group block rounded bg-none bg-transparent border border-transparent -m-px no-underline outline-none aria-current-page:border aria-current-page:border-primary aria-current-page:text-primary p-1"
           :aria-current="isCurrent(link.href) ? 'page': false"
+          data-breadcrumb-item
+          :data-breadcrumb-to="isCurrent(link.href) ? link.href : undefined"
```

Comment:
I don't see how this is working… I'm only looking the children of `Breadcrumb` to look for these data attributes